src/test/java/com/solvd/micro9/users/messaging/UserProducerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.solvd.micro9.users.messaging;
+
+import com.google.gson.Gson;
+import com.solvd.micro9.users.domain.aggregate.User;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.kafka.test.EmbeddedKafkaBroker;
+import org.springframework.kafka.test.context.EmbeddedKafka;
+import org.springframework.kafka.test.utils.KafkaTestUtils;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Slf4j
+@SpringBootTest
+@DirtiesContext
+@EmbeddedKafka(partitions = 1, brokerProperties = {"listeners=PLAINTEXT://localhost:9092", "port=9092"})
+public class UserProducerTest {
+
+    private final static String TOPIC = "syncMongoElastic";
+
+    @Autowired
+    private EmbeddedKafkaBroker embeddedKafka;
+
+    @Autowired
+    private UserProducer producer;
+
+    @Test
+    public void verifyMessageSentToKafkaTest() {
+        //given
+        User user = new User("1111", "Liza", "Ya", "email@gmail.com", false);
+
+        Map<String, Object> consumerProps = KafkaTestUtils.consumerProps("test-group", "true", embeddedKafka);
+        try (Consumer<String, User> consumer = new KafkaConsumer<>(consumerProps)) {
+            consumer.subscribe(Collections.singleton(TOPIC));
+
+            //when
```

Comment:
@eyakauleva comments should be removed

Reply:
this is my approach for test writing which is much more readable

Reply:
@eyakauleva your tests should be as your main source code
no empty lines and no comments

if you need comments to describe what's going on then probably your test code is low quality, fix this!

imagine you have at least 100 test classes, I don't think it will be comfortable to read all the comments, and try to understand it

and, if somebody notice some problems with your test mostly it will be disabled or 
even deleted

WDYT?